simcon_project/conversation_templates/forms.py

```diff
@@ -20,6 +21,10 @@ class FolderCreationForm(BSModalModelForm):
     """
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        # Only show templates unique to each user in the form.
+        if self.request.user:
+            templates = ConversationTemplate.objects.filter(researcher=self.request.user.id)
+            self.fields['templates'].queryset = templates.all()
```

Comment:
Is there a specific reason to call `.all()`?

Reply:
That's true, it probably works without it since I'm already using filter in the line above it